rmldatatfbbox/rmldatatfbbox/core.py

```diff
@@ -5,77 +5,121 @@
 Core command group and commands for TF Bounding Box dataset plugin.
 """
 import click 
-
-### OPTIONS ###
-
+import sys
+import io
+import importlib
+import shutil
+from pathlib import Path
+from rmldatatfbbox.utils.helpers import BboxDatasetWriter
+from ravenml.utils.question import user_input, cli_spinner
+from ravenml.data.helpers import default_filter_and_load
+from ravenml.data.options import pass_create
+from ravenml.data.interfaces import CreateInput, CreateOutput
 
 ### COMMANDS ###
-@click.command(help='TensorFlow Object Detection with bounding boxes.')
+@click.group(help='TensorFlow Object Detection with bounding boxes.')
```

Comment:
nit, but if there's only one command, we should maybe consider forgoing the group. I find the `ravenml data create tf-bbox create` syntax a bit confusing compared to just `ravenml data create tf-bbox`.

Reply:
Yeah, that was something I was thinking as well

Reply:
I've thought about this a lot and decided against this simplification for one reason:

While `ravenml data create tf-bbox create` is unwieldy compared to `ravenml data create tf-bbox`, it ensures a consistent interface across plugins that may have > 1 command. This was particularly apparent on the training side where multiple plugins already have > 1 command. I think it is more confusing to a user to jump between plugins defined as a single command vs. plugins defined as a command group than it is to just type one extra word.

Caveat to this in light of https://github.com/autognc/ravenML/pull/43: because of lazy loading, autocompletion (which we can and should add to ravenML) will no longer work for plugins. Thus maybe it is more convenient to just let plugins choose to be a group OR a single command, as users will not be able to autocomplete that pesky extra `create`.